lib/bamboo/email.ex

```diff
@@ -117,7 +117,12 @@ defmodule Bamboo.Email do
 
   for function_name <- @address_functions do
     @doc """
-    Sets the #{function_name} on the email
+    Sets the `#{function_name}` on the email
+
+    ```
+    new_email
+    |> #{function_name}(["sally@example.com", "james@example.com"])
```

Comment:
> We've been indenting by 4 spaces instead of using ```. I think after that's changes this can be merged in :D


Uh...you sure about that? https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/email.ex#L21-L34

https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/email.ex#L42-L67